copy/copy.go

```diff
@@ -122,10 +124,13 @@ func Image(policyContext *signature.PolicyContext, destRef, srcRef types.ImageRe
 		}
 	}()
 
-	// Please keep this policy check BEFORE reading any other information about the image.
-	if allowed, err := policyContext.IsRunningImageAllowed(unparsedImage); !allowed || err != nil { // Be paranoid and fail if either return value indicates so.
-		return errors.Wrap(err, "Source image rejected")
+	if policyContext != nil {
+		// Please keep this policy check BEFORE reading any other information about the image.
+		if allowed, err := policyContext.IsRunningImageAllowed(unparsedImage); !allowed || err != nil { // Be paranoid and fail if either return value indicates so.
+			return errors.Wrap(err, "Source image rejected")
+		}
```

Comment:
I’d rather force the caller to provide a policy. It is… not _too_ difficult… to do:
```go
signature.Policy{Default: []signature.PolicyRequirement{signature.NewPRInsecureAcceptAnything()}}
```
and passing `nil` here by mistake could be disastrous.